SPECS/nmi/nmi.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+ "Signatures": {
+  "nmi-1.7.0-vendor.tar.gz": "6addfc0548e96b65d3cb49f6f45ab4e9d42b854f4ba9eca524f257861b6b1374",
```

Comment:
I think you've accidentally checked-in the vendor tarball into the repo. Please remove it.